src/govuk/components/modal-dialogue/_modal-dialogue.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+@include govuk-exports("govuk/component/modal-dialogue") {
+  $govuk-dialogue-width: 640px;
```

Comment:
We probably want to use [`!default`](https://sass-lang.com/documentation/variables#default-values) here so that the value can overridden.